.github/workflows/run-test-cases.yml

```diff
@@ -84,7 +109,22 @@ jobs:
       run: |
         python -m pip install --upgrade pip
         pip install kubernetes
-    
+
+    - name: Build local containers for PR tests
+      if: startsWith(github.event_name, 'pull_request')
+      env:
+        BUILD_AMD64: 1
+        BUILD_ARM32: 0
+        BUILD_ARM64: 0 
+        PREFIX: ghcr.io/deislabs/akri
+        LABEL_PREFIX: pr
+      run: |
+        make akri-build
```

Comment:
Should we delete the images at the end of the test, so we dont have a bunch of pr image in our container registry?